asset/verifier.go

```diff
@@ -33,7 +33,7 @@ func (v *Sha512Verifier) Verify(rs io.ReadSeeker, desiredSHA string) error {
 	}
 
 	if foundSHA := hex.EncodeToString(h.Sum(nil)); foundSHA != desiredSHA {
-		return fmt.Errorf("sha512 '%s' does not match specified sha512: '%s'", desiredSHA, foundSHA)
+		return fmt.Errorf("sha512 of downloaded asset (%s) does not match specified sha512 in asset definition (%s)", foundSHA, desiredSHA)
 	}
```

Comment:
Since you're working on this section of code already, do you mind fixing https://github.com/sensu/sensu-go/issues/4064 ?

Reply:
Just pushed https://github.com/sensu/sensu-go/pull/4118/commits/0f5af5329833fc6500a2b8c974609e3b54a2b8cd